content/en/about/bugs/index.md

```diff
@@ -26,24 +26,20 @@ to learn what to do.
 
 If you're running on Kubernetes, consider including a cluster state
 archive with your bug report.
-For convenience, you can run a dump script to produce an archive containing
+For convenience, you can run a `bug-report` command to produce an archive containing
 all of the relevant state from your Kubernetes cluster:
 
-* Run via `curl`:
-
     {{< text bash >}}
-    $ curl {{< github_file >}}/tools/dump_kubernetes.sh | sh -s -- -z
+    $ istioctl bug-report
     {{< /text >}}
 
-* Run locally, from the release directory's root:
-
-    {{< text bash >}}
-    $ @tools/dump_kubernetes.sh@ -z
-    {{< /text >}}
+Then attach the produced `bug-report.tgz` with your reported problem.
 
-Then attach the produced `istio-dump.tar.gz` with your reported problem.
+    {{< warning >}}
+    `istioctl bug-report` command is only available with `istioctl` version `1.8.0` and higher which can be used against an older Istio version installed in your cluster to collect this information.
+    {{< /warning >}}
```

Comment:
```suggestion
{{< tip >}}
The `istioctl bug-report` command is only available with `istioctl` version `1.8.0` and higher but it can be used to also collect the information from an older Istio version installed in your cluster.
{{< /tip >}}
```